src/match/api.js

```diff
@@ -1,7 +1,8 @@
 import { setByPath } from '../firebase/initialize';
 import { getDataFromGoogleSpreadsheet } from '../googleDriveApi/getData';
 
-const surveyQuestionsDriveId = process.env.REACT_NATIVE_SURVEY_QUESTIONS_DRIVE_ID;
+//const surveyQuestionsDriveId = process.env.REACT_NATIVE_SURVEY_QUESTIONS_DRIVE_ID;
```

Comment:
I'm guessing this was greg's env variable stuff.  Do we maybe want an env file with defaults that are overwritten by process vars? or even override it here using `process.env.REACT_NATIVE_SURVEY_QUESTIONS_DRIVE_ID || '1Kh36D5DOyO8_it63OD-o51b2F4rlKfeLwnCbnaQZ5XE'`?

Reply:
I think so yeah. We could, and I guess we would also be doing the same for the firebase initialize? But I think that's how it used to be, so I'm not sure if there's any cons to that way of doing it.

Reply:
I just want to make sure that what I posted in slack this morning makes sense with respect to environment variables

Reply:
Yup! That was clear!